boards/native/common/natsim_config.cmake

```
@@ -13,6 +13,10 @@ if(SYSROOT_DIR)
   target_link_options(native_simulator INTERFACE "--sysroot=${SYSROOT_DIR}")
 endif()
 
+if("${LINKER}" STREQUAL "lld")
+	target_link_options(native_simulator INTERFACE "-fuse-ld=lld")
+endif()
```

Indent using two spaces

```suggestion
if("${LINKER}" STREQUAL "lld")
  target_link_options(native_simulator INTERFACE "-fuse-ld=lld")
endif()
```